qiskit/providers/aer/profile.py

```diff
@@ -0,0 +1,387 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Profile backend options for optimal performance
+"""
+import configparser as cp
+import os
+from socket import gethostname
+import numpy as np
+
+from qiskit import transpile, assemble, execute, user_config, QuantumRegister, QuantumCircuit
+from qiskit.circuit.library import QuantumVolume
+from qiskit.quantum_info import random_unitary
+from .aererror import AerError
+
+
+def profile_performance_options(min_qubits=10, max_qubits=25, ntrials=5,
```

Comment:
This `profile.py` file should probably be moved into `aer/utils/` and the user facing functions added to `aer/utils/__init__.py`